bucket/opentabletdriver.json

```
@@ -0,0 +1,48 @@
+{
+    "version": "0.6.0.4",
+    "description": "Open source, cross-platform, user-mode tablet driver",
+    "homepage": "https://opentabletdriver.net",
+    "license": {
+        "identifier": "LGPL-3.0",
+        "url": "https://github.com/OpenTabletDriver/OpenTabletDriver/blob/master/LICENSE"
+    },
+    "suggest": {
+        "SDK": "versions/dotnet6-sdk",
+        "WinUSB": "nonportable/zadig-np"
+    },
+    "notes": "Only certain tablets require zadig, see https://opentabletdriver.net/Wiki/Install/Windows#winusb.",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/OpenTabletDriver/OpenTabletDriver/releases/download/v0.6.0.4/OpenTabletDriver.win-x64.zip",
+            "hash": "4a3757df33e7f628839428c7093c1957c504c12429205cdf8ba957453c7104a8"
+        }
+    },
+    "bin": [ 
+        "OpenTabletDriver.UX.Wpf.exe",
+        "OpenTabletDriver.Daemon.exe"
+    ],
+    "shortcuts": [
+        [
+            "OpenTabletDriver.UX.Wpf.exe",
+            "OpenTabletDriver"
+        ],
+        [
+            "OpenTabletDriver.Daemon.exe",
+            "OpenTabletDriver Daemon"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/OpenTabletDriver/OpenTabletDriver"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/OpenTabletDriver/OpenTabletDriver/releases/download/v$version/OpenTabletDriver.win-x64.zip"
+            }
+        },
+        "hash": {
+            "url": "$url.sha256sum"
+        }
```

```suggestion
        }
```
`$url.sha256sum` resolves to "https://github.com/OpenTabletDriver/OpenTabletDriver/releases/download/v$version/OpenTabletDriver.win-x64.zip.sha256sum".

Therefore, this will only work if the checksum file ".sha256sum" exists.

However there is no such file, so we should remove `autoupdate.hash` section, and let *Scoop* get hash by downloading the ZIP file.

---

Ohh, thank you. I was a bit confused on how hashing worked for this haha.